INSTALL_ASU.md

```diff
@@ -0,0 +1,63 @@
+# General
+
+Start with the general [Enterprise new user setup](https://docs.google.com/document/d/1DOoiYEZd15KMw4KNSIkaGnph2aARI4n77cEuQHiOJqs/edit), and do the steps up to and including anaconda installation. When you create your environment, **be sure to use python 2**, not 3 as is the default. Give an appropriate name to the conda environment. Below we use `epic` as the example.
+
+
+After the installation of anaconda do the following
+```
+$ user=`whoami`
+$ env=epic
+ $ conda create -n ${env} python=2.7
+```
+It is advised to install the dependencies of LSL before attempting a `pip install`. While pip generally is good about grabbing package dependencies, the configuration of some packages requires dependencies to be installed before pip can check if the dependencies are installed.
+
+While not explicitly mentioned, lsl depends on `astropy` for fits I/O. Inclusion of astropy is not a mistake.
+
+To install dependencies for lsl excute the following lines. This may not be necessay depending on your environment but will help smooth the install.
+```
+$ conda activate ${env}
+$ conda config --add channels conda-forge
```

Comment:
```suggestion
$ conda install -c anaconda atlas
$ conda config --add channels conda-forge
```

Reply:
I'll go ahead and throw this out there:  do you want to switch to the "new" LSL?

Reply:
sure? How does one do that?

Reply:
It should be as easy as `pip install git+https://github.com/lwa-project/lsl.git` as long as you have libfftw3.  The caveats are: 
 1.  it has an incompatible API with the current LSL that LWA_EPIC uses (but there's a branch for that) and
2. it's technically not released yet.

Reply:
Ok, I think since there's an API change we should update the install instructions with the branch you're working on.